CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### New features
+
+- [#2118: Make the manage prototype pages independent of the govuk-frontend plugin](https://github.com/alphagov/govuk-prototype-kit/pull/2118)
```

Comment:
I think it might be more of a fix - this doesn't really introduce new functionality, it fixes a potential issue

Reply:
Moved change log comment as requested